### PR TITLE
Merl 586 finalize new faust website

### PR DESCRIPTION
**Pull request by TeresaGobble**
## Description

- Configured Sidebar
- Evaluated intro text on homepage (did not change this, as it is still accurate)
- Updated “features” on homepage to add Apollo
- Migrated Blog
- Removed lingering docs that don’t apply to new Faust
- Removed “edit this page” links from docs
- Updated twitter link to connect to the wpengine twitter 
- Fixed bug with who the article is by and when it was published (it was related to the production command in Atlas, which will run `npm run start` as the production branch, whereas this was the dev branch in the `config`'s script)

## Testing

Previews should show the site's changes and provide you with an opportunity to click through to ensure it is working. If not:

rom the root, cd into the `faustjs.org` folder.
In the command line, run `npm run build`
run `npm run dev'

### CODE REVIEW

**Comment by theodesp** [comment]
Hey @TeresaGobble great job here. I've noticed the following issues with some pages:

1. Custom post types link points to a 404 page:
Screenshot 2022-11-02 at 11 37 21
2. React hooks page point to old docs referencing GQty hooks. We should remove the link and the relevant docs from there:
Screenshot 2022-11-02 at 11 37 50

Screenshot 2022-11-02 at 11 38 47

3 Next.js, GraphQl and SSR/SSG links all point to 404 pages. We should either remove any links or change the links to a valid documentation page.

4. Both Tutorial and Getting started Links on the footer point to a 404 page:
Screenshot 2022-11-02 at 11 42 06

**Comment by blakewilson** [comment]
Nice work @TeresaGobble! let's also refine the menu a bit since with removing `Usage with Next.js` and just put everything at the top level. Something like:

```
- Getting Started
- Example Project Walkthrough
- Guides
- - Previews
- Reference
- - Seed Query
- - API Router
- - getNextServerSideProps
- - getNextStaticProps
- Companion WordPress Plugin
- - Settings Reference
- Telemetry
- Migration from old Faust
```

**Comment by TeresaGobble** [comment]
@theodesp and @blakewilson , thank you so much for this detailed feedback! Will do!

**Comment by TeresaGobble** [comment]
Thank you @theodesp and @blakewilson for your feedback! I have: 

Per Theo's feedback,
1. Deleted the Custom Post Types feature card from the front page
2. Deleted the React Hooks card from the front page
3. Deleted the Next.js, GraphQL and SSR/SSG cards from the front page
4. Updated the `getting started` link and deleted the `tutorial` link.

Per Blake's feedback, updated the sidebar menu tree.
image

Additionally, I added a seed query and nextjs card for the `getNextServerSideProps` function. Please let me know if you think other cards should be added! I added the example project to the footer as well.

image

**Comment by theodesp** [comment]
Hey Nice work here @TeresaGobble .

I just went through all the documentation site checking the content.  I found two small inconsistencies:

1. In the Using the API Router page, the code examples mention the old Faust code: `faustjs` instead of `faustwp`
Screenshot 2022-11-03 at 11 24 50
2. The Migration Path page is empty. Maybe add a NOTE message for WORK IN PROGRESS.
Screenshot 2022-11-03 at 11 27 42

File: internal/faustjs.org/docs/migrationPath/overview.mdx [hunk 1]
```diff
@@ -1,4 +1,7 @@
 ---
 slug: /migrationPath/overview
-title: Migrate from old version of Faust.js ( < v1.0.0 ) to Faust ( > v1.0.0 )
+title: Migration from Old Faust
```
Comment (josephfusco) [inline internal/faustjs.org/docs/migrationPath/overview.mdx:3]:
```suggestion
title: Migration from Legacy Faust
```

File: internal/faustjs.org/sidebars.js [hunk 2]
```diff
@@ -91,48 +80,9 @@ module.exports = {
...
-      label: 'Migration Path from Faust.js < v1.0.0',
-      items: [
-        {
-          type: 'doc',
-          label: 'Overview',
-          id: 'migrationPath/overview',
-        },
-      ],
+      type: 'doc',
+      label: 'Migration from old Faust',
```
Comment (josephfusco) [inline internal/faustjs.org/sidebars.js:84]:
```suggestion
      label: 'Migration from legacy Faust',
```

**Review by josephfusco — APPROVED** [review]
Great job on this!

Wondering if we should update "Old Faust" to "Legacy Faust"?

**Pull request merged**